.github/workflows/go-ci.yaml

```diff
@@ -0,0 +1,114 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+---
+name: Go Continuous Integration
+
+on:
+  workflow_call:
+    secrets:
+      gh_username:
+        required: false
+      gh_token:
+        required: false
```

Comment:
Can't we just use the in-built token concept? https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication